static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -111,14 +121,23 @@ function GroupRelatedIssues({params}: Props) {
                 <Title>{t('Trace connected issues')}</Title>
                 {traceConnectedIssues.length > 0 ? (
                   <div>
-                    <small>
-                      {t('These are the issues belonging to ')}
-                      <Link
-                        to={`/performance/trace/${traceMeta.trace_id}/?node=error-${traceMeta.event_id}`}
+                    <TextButtonWrapper>
+                      <small>
+                        {t('These are the issues belonging to ')}
+                        <Link
+                          to={`/performance/trace/${traceMeta.trace_id}/?node=error-${traceMeta.event_id}`}
+                        >
+                          {t('this trace')}
+                        </Link>
+                      </small>
+                      <LinkButton
+                        href={`/issues/?query=issue.id:[${groupId},${traceConnectedIssues}]`}
```

Comment:
You should be able to just link to the trace directly with the `trace:` operator here, ex: https://sentry.sentry.io/issues/?project=1&query=trace%3A441502f3c47a4aedb5bc6fe7e29f7a52.